lms/static/scripts/frontend_apps/components/BookSelector.tsx

```diff
@@ -128,8 +128,6 @@ export default function BookSelector({
     // Focus the input field when the component is first rendered
     inputRef.current!.focus();
     // We only want to run this effect once.
-    //
-    // eslint-disable-next-line react-hooks/exhaustive-deps
   }, []);
```

Comment:
The line above about running the effect once was part of / about this eslint override.